pages/edge/learn/connecting-domains-to-edge.mdx

```diff
@@ -0,0 +1,65 @@
+import { Callout } from "nextra-theme-docs";
+
+# Connecting your domain to Wasmer Edge
+
+## Introduction
+
+For any functional website/server, you need a domain name. This is the address that people will use to access your services.
```

Comment:
This is misleading, you can use Edge just fine with the `*.wasmer.app` domains.

Can we rephrase this to something more like "For production workloads, you most likely want to use your own custom domain, instead of the *.wasmer.app domains. This guide will explain how to connect your custom domains to your apps..."